src/renderer/components/Wallet/WalletSidebar/WalletSidebar.vue

```diff
@@ -195,11 +195,14 @@ export default {
   }),
 
   computed: {
+    fetchContactsOnly () {
```

Comment:
I'd rename it to `hasContactsOnly`, `includesContactsOnly` or something like that, using "fetch" looks like that method / computed property request the data.

Reply:
i changed it to `hasContactsOnly`